src/tests/suite/daos_test_common.c

```diff
@@ -930,6 +930,22 @@ get_daos_acl_with_owner_perms(uint64_t perms)
 	return acl;
 }
 
+daos_prop_t *
+get_daos_prop_with_owner_and_acl(char *owner, uint32_t owner_type,
+				 struct daos_acl *acl, uint32_t acl_type)
+{
+	daos_prop_t	*prop;
+
+	prop = daos_prop_alloc(2);
```

Comment:
is an assertion that prop != NULL needed in this and similar functions, in case the allocation fails?

Reply:
Good idea, fixing.